### PR TITLE
feat(MangaGo): add new presence

### LINKED ISSUE #7786: MangaGo | www.mangago.me
### Website name

MangaGo

### Website URL

https://www.mangago.me/

### Website logo

https://pic1.mangapicgallery.com/images/ywz-2-7070.png?1

### Prerequisites

- [ ] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

Since it is a manga website, it could have the same or similar layout as other manga websites that have already have a presence. 

For what should be displayed, it should show what manga & what chapter out of the entire series is being read by the user.  For example, it should show up as:

"Reading [Insert title here]..."
"Reading Chapter 15 out of 65"

### PR DESCRIPTION
**Pull request by Hikki0710**
## Description 
Resolves #7786 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image
image
image
image

with privacy enabled
image
image
image

</details>

### CODE REVIEW

File: websites/M/MangaGo/metadata.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"author": {
+		"id": "533014724569333770",
+		"name": "_hikki_"
+	},
+	"service": "MangaGo",
+	"description": {
+		"en": "MangaGo is a popular online platform for reading manga, manhwa and more, offering a vast library of titles across various genres",
+		"it": "MangaGo è una popolare piattaforma online per la lettura di manga, manhwa e altro ancora, che offre una vasta libreria di titoli di vari generi"
+	},
+	"url": "mangago.me",
```
Comment (theusaf) [inline websites/M/MangaGo/metadata.json:12]:
```suggestion
	"url": ["www.mangago.me","mangago.me"],
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Comment by Hikki0710** [comment]
> Images kinda blurry can you improve that?

Unfortunately the logos and thumbnail are from the official website and they only have a resolution of 70x70 (so they become blurry when I change them to 512x512 with [imageresizer](https://imageresizer.com/)). Could I change the thumbnail using a screenshot from the website?

https://i.imgur.com/1YR03XH.png

image

**Comment by theusaf** [comment]
> > Images kinda blurry can you improve that?
> 
> Unfortunately the logos and thumbnail are from the official website and they only have a resolution of 70x70 (so they become blurry when I change them to 512x512 with [imageresizer](https://imageresizer.com/)). Could I change the thumbnail using a screenshot from the website?

Yes, the thumbnail can be a screenshot of the website. For the logo, you could try using tools like [waifu2x](http://waifu2x.udp.jp/) as recommended in the guidelines, converting it to an svg, or other similar methods.

**Comment by Hikki0710** [comment]
okay they look good to me

**Review by Bas950 — APPROVED** [review]

**Pull request merged**